swap/src/bitcoin/timelocks.rs

```diff
@@ -49,6 +49,14 @@ impl Add<Timelock> for BlockHeight {
     }
 }
 
+impl Sub<BlockHeight> for BlockHeight {
+    type Output = BlockHeight;
+
+    fn sub(self, rhs: BlockHeight) -> Self::Output {
+        BlockHeight(self.0 - rhs.0)
+    }
+}
+
```

Comment:
Please clarify why you would substract two absolute block heights?

Also, implementing `sub_checked` would be safer.

Reply:
Yep switched to checked-sub this morning. Haven't pushed it.

We are subtracting the transaction block height with the latest block height to get the number of confirmations. BDK wallet does not have a API for this.